docker/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  postgres:
+    image: postgres:latest
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: postgres
```

Comment:
I assume this is the default password? We should use a unique password in the example. Maybe someone will see a real password and decide for themselves not to use the default. Saving the world, avoiding one default password at a time. :)

Reply:
Sure, I'll give it the l33tc0de treatment

Reply:
Hahhaha

Reply:
n00bDB@1